File: Java/1284.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int cnt = 0;
		int idx = 0;
		int arr[] = new int[2];
		int n = sc.nextInt();
		
		for(int i = 2; i < n; i++){
			if(n%i==0){
				cnt++;
				
				if(i==2 && idx < 2){
					arr[idx] = i;
					idx++;
				} else if(i%2!=0 && idx < 2){
					arr[idx] = i;
					idx++;
				}
			}
		}
		
		if(cnt==2 && idx==2) System.out.printf("%d %d", arr[0], arr[1]);
		else System.out.printf("wrong number");
		
	}
}

File: Python/1284.py
n = int(input())
cnt, idx = 0, 0
arr = [0, 0]

for i in range(2, n):
    if n % i == 0:
        cnt += 1

        if i==2 and idx < 2:
            arr[idx] = i
            idx += 1
        elif i%2!=0 and idx < 2:
            arr[idx] = i
            idx += 1

if cnt==2 and idx==2 : print("%d %d" % (arr[0], arr[1]))
else : print("wrong number")